### ISSUE TITLE
Accessibility: Glossary is missing a form label

### ISSUE THREAD

**Issue by ecayer** [GSA/digitalgov-pra#173]
According to WAVE, the search box is missing a form label
<img width="382" alt="Screen Shot 2019-05-10 at 9 47 24 AM" src="https://user-images.githubusercontent.com/3640935/57543238-b11e5b00-7308-11e9-945b-247b86d7ad4a.png">

**Comment by hursey013** [comment]
Fixed with https://github.com/GSA/digitalgov-pra/commit/48b6cfb62a063f57c432799bab763cfd02b26f17
